src/components/views/right_panel/WidgetCard.tsx

```diff
@@ -40,19 +40,19 @@ const WidgetCard: React.FC<IProps> = ({ room, widgetId, onClose }) => {
 
     const apps = useWidgets(room);
     const app = apps.find(a => a.id === widgetId);
-    const isPinned = app && WidgetLayoutStore.instance.isInContainer(room, app, Container.Top);
+    const isRight = app && WidgetLayoutStore.instance.isInContainer(room, app, Container.Right);
```

Comment:
previously (~a year) there were issues with Jitsi widgets accidentally ending up in the right panel, so the check was changed to detecting pinned state instead. 

I can believe we've fixed the issue since, but would be good to double check that 1:N conference calls don't end up in the right panel by accident.

Reply:
Hmm, I tested the Jitsi conference path and wasn't able get them stuck anywhere... They start out as pinned and can be moved to other containers like any other widget, so I think this is probably safe.